testfiles/FurnaceWithDXSystem_CoolingHXAssisted.idf

```diff
@@ -960,7 +957,10 @@
     For: AllOtherDays,       !- Field 13
     Until: 6:00,1.0,         !- Field 14
     Until: 13:00,0.0,        !- Field 16
-    Until: 24:00,1.0;        !- Field 18
+    Until: 24:00,1.0,        !- Field 18
+    Through: 12/31,          !- Field 1
+    For: AllDays,            !- Field 2
+    Until: 24:00,0.0;        !- Field 3
```

Comment:
This file also shows diffs, and for the same reason, this schedule was completely wrong before.